src/rf/apps/workers/process.py

```diff
@@ -7,10 +7,15 @@
 import apps.core.enums as enums
 from django.conf import settings
 
+import os
 import time
+import uuid
 import json
 import boto.sqs
 from boto.sqs.message import Message
+from boto.s3.key import Key
```

Comment:
I am concerned about the mixing of `boto` and `boto3` in this module. We already had some `boto` stuff to support uploads, and porting that to `boto3` doesn't seem straightforward, but in this module things are meant to flow from one function to another. It will probably get difficult to debug if one step is `boto` and the other is `boto3`.

(I think the EMR step is being implemented in `boto3`.)

/cc @kdeloach